### DISCUSSION TITLE
[Feature Request] Support for Additional Casing Configuration Formats

### DISCUSSION THREAD

**Discussion by magic-akari** [biomejs/biome#1896] [General]
At present, Biome only supports `camelCase` as the configuration format. 
It would be beneficial to extend this support to include other formats like `snake_case`.

This enhancement would prove valuable for downstream projects that rely on diverse casing conventions.

Here is a solution on how to support multiple casings: https://github.com/g-plane/markup_fmt/pull/7/files

**Comment by ematipico** [comment]
I'm not sure I see **enough value** in implementing this in Biome. We have a massive configuration, our rules are named in `camelCase` across website, configuration, diagnostics and suppression comments. It will take a massive amount work for us.

**Comment by Conaclos** [comment]
Are you using `biome_deserialize` macro? Or are you suggesting that the biome config file should support several cases?
